bukuserver/templates/bukuserver/bookmarks_list.html

```diff
@@ -6,9 +6,22 @@
   {{ buku.close_if_popup() }}
 {% endblock %}
 
+{% block model_menu_bar_before_filters %}
+  {{ super() }}
+  {% if data %}
```

Comment:
The Random button is not displayed when the list is empty (to avoid dealing with error 404)